Cargo.lock

```diff
@@ -1735,9 +1735,9 @@ dependencies = [
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.51"
+version = "1.0.69"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "5d727cae5b39d21da60fa540906919ad737832fe0b1c165da3a34d6548c849d6"
+checksum = "134c189feb4956b20f6f547d2cf727d4c0fe06722b20a0eec87ed445a97f92da"
```

Comment:
Are the dependency updates in Cargo.lock necessary? We like to keep dependency updates separate from other changes and only add if needed.

Reply:
Alright noted I believed since `Cargo.lock` was put into for version update but need some further info upon that.